qcfractal/cli/qcfractal_manager.py

```diff
@@ -12,16 +16,136 @@
 
 __all__ = ["main"]
 
+QCA_RESOURCE_STRING = '--resources process=1'
+MANAGER_CONFIG_NAME = "qcf_manager_config.yaml"
+
+
+class SettingsCommonConfig:
+    env_prefix = "QCA_"
+    case_insensitive = True
+
+
+class AdapterEnum(str, Enum):
+    dask = "dask"
+    pool = "pool"
+
+
+class CommonManagerSettings(BaseSettings):
+    ntasks: int = 1
+    cores: int = qcng.config.get_global("ncores")
+    memory: confloat(gt=0) = qcng.config.get_global("memory")
+    max_tasks: conint(gt=0) = 200
+    manager_name: str = "unknown"
+    update_frequency: float = 30
+    test: bool = False
+    adapter: AdapterEnum = AdapterEnum.pool
+    log_file_prefix: str = None
+    queue_tag: str = None
+
+    class Config(SettingsCommonConfig):
+        pass
+
+
+class FractalServerSettings(BaseSettings):
+    file: str = None
```

Comment:
Or we just assume we try to read and overwrite from cli, its what most programs which use a `~/.{folder}/config` file do anyways, no need to make it more complex, especially since that means we would have to engineer a `False` condition